hasura/metadata/databases/carnet_de_bord/tables/public_structure.yaml

```
@@ -52,7 +52,7 @@ computed_fields:
       function:
         name: nb_beneficiary_for_structure
         schema: public
-    comment: "a computed field, executes function nb_beneficiary_for_structure "
+    comment: 'a computed field, executes function nb_beneficiary_for_structure '
```

Ce changement ne semble pas avoir de rapport avec le sujet de la PR ?
```suggestion
    comment: "a computed field, executes function nb_beneficiary_for_structure "
```